### DISCUSSION TITLE
Error: Invalid hook call. Hooks can only be called inside of the body of a function component.

### DISCUSSION THREAD

**Discussion by SergiiMost** [vercel/next.js#14406] [Help] [answered]
Yesterday everything worked just fine. Today, without any code change i get this error. Probably the version of Next.js was updated. Is there an easy way to fix it? Thanks!

**Comment by nullish-dev** [comment]
If the code was working before,You can try deleting the .next folder and try running the server again.

**Reply by SergiiMost** [reply]
Just tried to do so, it didn't work. Thanks though.

**Comment by SergiiMost** [comment, marked as answer]
Ok,   i was able to fix the problem. Apparently, it was related to the **WindowsPowerShell** .  Here is [SO](https://stackoverflow.com/questions/58365151/hooks-error-invalid-hook-call-using-nextjs-or-reactjs-on-windows) post  that explains everything. I hope this helps someone else.
